stac-extension/README.md

```diff
@@ -0,0 +1,95 @@
+# Humanitarian OpenStreetMap Team OpenAerialMap Extension Specification
+
+- **Title:** Humanitarian OpenStreetMap Team OpenAerialMap (OAM) Extension
+- **Identifier:** <https://raw.githubusercontent.com/hotosm/stactools-hotosm/refs/heads/main/stac-extension/json-schema/schema.json>
+- **Field Name Prefix:** oam
+- **Scope:** Item
+- **Extension [Maturity Classification](https://github.com/radiantearth/stac-spec/tree/master/extensions/README.md#extension-maturity):** Proposal
+- **Owner**: @ceholden @gadomski
+
+This extension documents metadata used by the Humanitarian OpenStreetMap Team (HOT)'s OpenAerialMap (OAM) project.
+It builds on common STAC metadata by defining some optional common metadata as required attributes and defines
+the expected values for common metadata.
+
+- Examples:
+  - [Item example](./examples/item.json): Shows the basic usage of the extension in a STAC Item
+- [JSON Schema](./json-schema/schema.json)
+- [Changelog](./CHANGELOG.md)
+
+## Fields
+
+The fields in the table below can be used in these parts of STAC documents:
+
+- [ ] Catalogs
+- [ ] Collections
+- [x] Item Properties (incl. Summaries in Collections)
+- [ ] Assets (for both Collections and Items, incl. Item Asset Definitions in Collections)
+- [ ] Links
+- [x] Provider
+
+| Field Name        | Type   | Description                                                                                          |
+| ----------------- | ------ | ---------------------------------------------------------------------------------------------------- |
+| provider          | string | **REQUIRED**. The imagery data provider. The provider must also be included in the "provider" field. |
```

Comment:
The interaction with [`providers` from common metadata](https://github.com/radiantearth/stac-spec/blob/master/commons/common-metadata.md#provider) is awkward. Could we change to re-use `providers`? (apologies for not catching this sooner.

Reply:
No problem! 

I added this because searching by provider name is one of the frontend use case requirements, and AFAIK PgSTAC wants to work on Item properties and not other fields. The plurality of `providers` also makes this a little tricky, e.g., this SQL is not so nice,

```sql
select *
from pgstac.items
where collection = 'openaerialmap'
  and jsonb_path_query_array(content->'providers', '$[*].name') ? 'test user foo'
```

I can ask elsewhere but I can't figure out how that would work behind the scenes as a PgSTAC queryables or from frontend via CQL2 filter

Alternatively, would something like `producer_name` be clearer? It's trying to reference the name of the `providers` entry with the `producer` role

Reply:
😬 great point on the searchability. `oam:producer_name` feels like the nice compromise. Might be worth asking internally to see if folks have run into this before ... I hadn't thought of that.

Reply:
Seems like CQL2 support is the more tricky/impossible bit than support in Postgres, so we're stuck with an Item property

I updated the schema and added some more tests in https://github.com/hotosm/stactools-hotosm/pull/6/commits/70cec3fa3b6443d97de74a6736c4375111cc7269
I really like having the test there to triple check my jsonschema, didn't feel enough that our example "passed" and it'd be nice to have some counter-examples of an Item failing